### PR TITLE
refactor schema

### PR DESCRIPTION
**Pull request by BobBorges**
.

### CODE REVIEW

File: data/199495/ipq-199495--0005.xml [hunk 1]
```diff
@@ -3,76 +3,100 @@
...
-          <p>No correction of source texts was performed.</p>
-        </correction>
-      </editorialDecl>
-    </encodingDesc>
+    <profileDesc>
+      <particDesc>
+        <listPerson>
+          <person>
+            <persName>Sten Andersson (m)</persName>
+            <state ref="unknown" type="partyAffiliation">
```
Comment (ninpnin) [inline data/199495/ipq-199495--0005.xml:24]: Are we going to map m -> moderaterna or their ID at some point?

File: data/199495/ipq-199495--0007.xml [hunk 2]
```diff
@@ -3,73 +3,97 @@
...
-    </encodingDesc>
+    <profileDesc>
+      <particDesc>
+        <listPerson>
+          <person gender="woman">
+            <persName>Marie Wilén (c)</persName>
+            <state ref="unknown" type="partyAffiliation">
+              <desc>c</desc>
+            </state>
+            <idno>i-PZnqvmWZ9PhCRqw6uhJsKm</idno>
```
Comment (ninpnin) [inline data/199495/ipq-199495--0007.xml:28]: This is probably the best solution.

File: data/199495/ipq-199495--0008.xml [hunk 3]
```diff
@@ -3,88 +3,112 @@
   <teiHeader>
     <fileDesc>
       <titleStmt>
-        <title>Interpellation Question</title>
+        <title>miljöpolitiken vid ett EU-medlemskap</title>
```
Comment (ninpnin) [inline data/199495/ipq-199495--0008.xml:6]: Capitalize?

File: data/199495/ipq-199495--0009.xml [hunk 4]
```diff
@@ -3,93 +3,117 @@
...
+        <listPerson>
+          <person gender="man">
+            <persName>Per-Ola Eriksson (c)</persName>
+            <state ref="unknown" type="partyAffiliation">
+              <desc>c</desc>
+            </state>
+            <idno>i-GyYMjkDFkNwtQ7GJKMBnVE</idno>
+          </person>
+          <person>
+            <persName>finansministern</persName>
```
Comment (ninpnin) [inline data/199495/ipq-199495--0009.xml:31]: Later on, we want to map this to the actual person

**Review by ninpnin — APPROVED** [review]
LGTM

File: data/199495/ipq-199495--0009.xml (hunk 4 above)
Comment (BobBorges) [reply data/199495/ipq-199495--0009.xml:31]: that's the plan

**Pull request merged**